appveyor.yml

```diff
@@ -0,0 +1,25 @@
+version: "{build}"
+
+init:
+  - git config --global core.autocrlf input
+
+environment:
+  matrix:
+    # node.js
+    - nodejs_version: "0.10"
+    - nodejs_version: "0.11"
+    # io.js
+    - nodejs_version: "1.0"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+
+test_script:
+  - node --version
+  - npm --version
+  - npm install
```

Comment:
I think this line should be moved to install section (because it's about installation ;-)